src/driver/Firestore/InProcessFirestore.ts

```diff
@@ -1250,16 +1250,19 @@ class InProcessFirestoreBulkWriter implements IFirestoreBulkWriter {
     create(documentRef: IFirestoreDocRef<IFirestoreDocumentData>, data: IFirestoreDocumentData): Promise<IFirestoreWriteResult> {
         this.throwIfClosed()
 
-        this.writeOperations.push(async () => {
-            if ((await documentRef.get()).exists) {
-                throw new FirestoreError(
-                    GRPCStatusCode.ALREADY_EXISTS,
-                    `Document already exists: ${documentRef.path}`,
-                )
+        const createPromise = 
+            async () => {
+                if ((await documentRef.get()).exists) {
+                    throw new FirestoreError(
+                        GRPCStatusCode.ALREADY_EXISTS,
+                        `Document already exists: ${documentRef.path}`,
+                    )
+                }
+                await documentRef.set(data)
             }
-            await documentRef.set(data)
-        })
-        return this.mockIFirestoreWriteResult
+        const bulkWriteOpPromise = this.enqueue(createPromise)
+        silencePromise(bulkWriteOpPromise)
```

Comment:
I'm not sure why we need to ignore errors with these, either a comment explaining or some more explicit naming might help?

Reply:
+1, I only know why this is here because I looked at it with Sean